guiEditor/src/diagram/guiNode.ts

```diff
@@ -151,16 +165,53 @@ export class GUINode {
         this.x += newX;
         this.y += newY;  
 
+        this.children.forEach(child => {
+           child._onMove(evt, startPos, true);
+        });
+
         return true;
         //evt.stopPropagation();
     }
 
+    renderContainer(): React.ReactNode {
+        if(!this._isContainer) return null;
+
+        return React.createElement(ContainerPropertyTabComponent, {

```

Comment:
any reason this isn't a tsx file and render jsx/html fragement here?

Reply:
Not really. Originally this was paired besides the other render but since that function is gone maybe I should rethink this.